src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -843,383 +749,13 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
 // - parameters - set of numeric parameters collected while parsing the sequence.
 // Return Value:
 // - true iff we successfully dispatched the sequence.
-bool OutputStateMachineEngine::ActionSs3Dispatch(const wchar_t /*wch*/,
-                                                 const gsl::span<const size_t> /*parameters*/) noexcept
+bool OutputStateMachineEngine::ActionSs3Dispatch(const wchar_t /*wch*/, const VTParameters /*parameters*/) noexcept
 {
     // The output engine doesn't handle any SS3 sequences.
     _ClearLastChar();
     return false;
 }
 
-// Routine Description:
-// - Retrieves the listed graphics options to be applied in order to the "font style" of the next characters inserted into the buffer.
-// Arguments:
-// - parameters - The parameters to parse
-// - options - Space that will be filled with valid options from the GraphicsOptions enum
-// Return Value:
-// - True if we successfully retrieved an array of valid graphics options from the parameters we've stored. False otherwise.
-bool OutputStateMachineEngine::_GetGraphicsOptions(const gsl::span<const size_t> parameters,
-                                                   std::vector<DispatchTypes::GraphicsOptions>& options) const
-{
-    bool success = false;
-
-    if (parameters.empty())
-    {
-        options.push_back(DefaultGraphicsOption);
-        success = true;
-    }
-    else
-    {
-        for (const auto& p : parameters)
-        {
-            options.push_back((DispatchTypes::GraphicsOptions)p);
-        }
-        success = true;
-    }
-
-    // If we were unable to process the string, and there's a TTY attached to us,
-    //      trigger the state machine to flush the string to the terminal.
-    if (_pfnFlushToTerminal != nullptr && !success)
-    {
-        success = _pfnFlushToTerminal();
-    }

```

Comment:
Why the heck were we doing this here, and not in any of the other cases 🤔

Reply:
Technically it wasn't being done here either. This whole branch was a no-op, because success could never have been false.